### PR TITLE
Sec 3 identifiers identities credentials

### PR DESCRIPTION
**Pull request by Therecanbeonlyone1969**
# 3 Identifiers, Identity and Credential Management

### 3.2.2 BPI Identities and Credentials

Please, review and comment. This is a foundational section for a BPI.

### CODE REVIEW

File: core/baseline-core-v1.0-psd01.md [hunk 1]
```diff
@@ -566,179 +566,631 @@ Below we list and define the components of each layer. The detailed requirements
...
+Decentralized in this context means that there is no single point of failure in the PKI where possibly no participants are known to one another.
 
-| Requirement ID|Requirement  |
-| :--- | :--- |
-| COM8 | Durable messaging  |
-| COM9 | Persistent messaging|
-| COM10 | Latency  |
-| COM11 | Fault-tolerance |
-| COM12 | Scalability |
+**[R33]**	The PKI of a BPI MUST be strongly federated.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:598]: Any example or concrete number to be strongly federated?

File: core/baseline-core-v1.0-psd01.md [hunk 2]
```diff
@@ -566,179 +566,631 @@ Below we list and define the components of each layer. The detailed requirements
...
+
+Decentralized Public Key Infrastructure (DPKI) has been proposed as a secure alternative. The goal of DPKI is to ensure that, unlike PKIX, no single third-party can compromise the integrity and security of a system employing DPKI as a whole. 
+
+Within DPKI, a Principal Owner can be given direct control and ownership of a globally readable identifier by registering the identifier for example in a CCSM. Simultaneously, CCSMs allow for the assignment of arbitrary data such as public keys to these identifiers and permit those values to be globally readable in a secure manner that is not vulnerable to the MITM attacks that are possible in PKIX. This is done by linking an identifier’s lookup value to the latest and most correct public keys for that identifier. In this design, control over the identifier is returned to the Principal Owner. 
+
+Within this design. it is no longer trivial for any one entity to undermine the security of the entire DKPI system or to compromise an identifier that is not theirs overcoming the challenges of typical PKI.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:616]: typo `this design.`

File: core/baseline-core-v1.0-psd01.md [hunk 3]
```diff
@@ -566,179 +566,631 @@ Below we list and define the components of each layer. The detailed requirements
...
+
+Furthermore, DPKI requires a public registry of identifiers and their associated public keys that can be read by anyone but cannot be compromised. As long as this registration remains valid and the Principal Owner is able to maintain control of their private key, no 3rd party can take ownership of that identifier without resorting to direct coercion of the Principal Owner.
+
+**[O1]** A BPI MAY utilize a DPKI.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:620]: Agreed `should` is better

File: core/baseline-core-v1.0-psd01.md [hunk 4]
```diff
@@ -566,195 +568,642 @@ Below we list and define the components of each layer. The detailed requirements
...
+
+**[D10]** A credential utilized within one or more BPIs SHOULD be discoverable by a participant in said BPI(s).
+
+**[R50]** The presentation of a credential utilized within one or more BPIs MUST be cryptographically signed by the presenter of the credential, also known as the credential holder.
+
+See the [W3C Verifiable Credential Standard](#####-W3C-VC) for a definition of credential holder.
+
+**[R51]** If a credential holder is a BPI participant, the holder MUST have a unique identifier that has been established within the BPI context the holder operates in.
+
+As discussed in [Section 3.1](##31-Introduction-and-High-Level-Requirements), BPIs require either decentralized or strongly federated identifier/identity providers that have been agreed to by the participants in a BPI context of one or more BPIs.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:737]: remove `BPI` before `context` > `[...] in a context of one [...]`

File: core/baseline-core-v1.0-psd01.md [hunk 5]
```diff
@@ -566,195 +568,642 @@ Below we list and define the components of each layer. The detailed requirements
...
+**[R51]** If a credential holder is a BPI participant, the holder MUST have a unique identifier that has been established within the BPI context the holder operates in.
+
+As discussed in [Section 3.1](##31-Introduction-and-High-Level-Requirements), BPIs require either decentralized or strongly federated identifier/identity providers that have been agreed to by the participants in a BPI context of one or more BPIs.  
+
+<figure>
+  <img
+  src="./images/section-3-baseline-spec-identity-authorities.png"
+  >
+  <figcaption>Figure 5: Delineation of the Identity and Credential issuing authorities used in a BPI and their management within a BPI using Decentralized Identity Verifiers and OpenID Connect Relying Party as examples</figcaption>
+</figure>
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:744]: add `.` at the end of sentence?

File: core/baseline-core-v1.0-psd01.md [hunk 6]
```diff
@@ -566,195 +568,642 @@ Below we list and define the components of each layer. The detailed requirements
...
 
-This section describes mechanisms to ensure counterparties confidentiality and shielded private transactions.
+As depicted in Figure 5 above, the accepted Entity identity credentials or other credentials from Identity providers and presented by a BPI participant need to be verified by the BPI against the issuing providers and subsequently stored in the BPI, and if required, updated from the identity provider once expired, revoked or otherwise changed.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:746]: short explanation of each box or reference.

File: core/baseline-core-v1.0-psd01.md [hunk 7]
```diff
@@ -566,195 +568,642 @@ Below we list and define the components of each layer. The detailed requirements
...
+
+**[R51]** If a credential holder is a BPI participant, the holder MUST have a unique identifier that has been established within the BPI context the holder operates in.
+
+As discussed in [Section 3.1](##31-Introduction-and-High-Level-Requirements), BPIs require either decentralized or strongly federated identifier/identity providers that have been agreed to by the participants in a BPI context of one or more BPIs.  
+
+<figure>
+  <img
+  src="./images/section-3-baseline-spec-identity-authorities.png"
+  >
+  <figcaption>Figure 5: Delineation of the Identity and Credential issuing authorities used in a BPI and their management within a BPI using Decentralized Identity Verifiers and OpenID Connect Relying Party as examples</figcaption>
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:743]: short explanation of each box or reference.

**Review by Consianimis — APPROVED** [review]

File: core/baseline-core-v1.0-psd01.md (hunk 1 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: updated text to contextualize without giving hard number

File: core/baseline-core-v1.0-psd01.md (hunk 2 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: fixed

File: core/baseline-core-v1.0-psd01.md (hunk 3 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: updated

File: core/baseline-core-v1.0-psd01.md (hunk 4 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md:737]: updated

File: core/baseline-core-v1.0-psd01.md (hunk 5 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md:744]: updated

File: core/baseline-core-v1.0-psd01.md (hunk 7 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: added explanation

**Comment by Therecanbeonlyone1969** [comment]
Closing pull request since this PR is subsumed in PR #59.

**Pull request closed without merging**